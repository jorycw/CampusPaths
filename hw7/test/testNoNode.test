# Find a path with Dijkstra's algorithm when one of the nodes
# does not exist

CreateGraph graph

AddNode graph a
AddNode graph b
AddNode graph c

AddEdge graph a b 1
AddEdge graph b a 1
AddEdge graph b c 1
AddEdge graph c b 1

FindPath graph d c
FindPath graph c d
