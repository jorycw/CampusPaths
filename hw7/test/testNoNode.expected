# Find a path with Dijkstra's algorithm when one of the nodes
# does not exist

created graph graph

added node a to graph
added node b to graph
added node c to graph

added edge 1.000 from a to b in graph
added edge 1.000 from b to a in graph
added edge 1.000 from b to c in graph
added edge 1.000 from c to b in graph

unknown character d
unknown character d